crates/client/assets/schema.sdl

```diff
@@ -357,7 +359,6 @@ type GasCosts {
 	divi: U64!
 	ecr1: U64!
 	eck1: U64!
-	ed19: U64!
```

Comment:
We shouldn't remove old `ed19`. Because of backward compatibility we need to add `ed19DependentCost`. As we did for `aloc` opcode

Reply:
Should be fixed in ebe4c2111, let me know if that looks correct.